Testing/SystemTests/tests/framework/CorelliPowderCalibrationTest.py

```diff
@@ -4,14 +4,36 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 #pylint: disable=no-init
 from systemtesting import MantidSystemTest
-from mantid.simpleapi import mtd
-from mantid.simpleapi import CreateEmptyTableWorkspace
-from mantid.simpleapi import LoadEmptyInstrument
-from mantid.simpleapi import ConvertToEventWorkspace
-from mantid.simpleapi import CompareWorkspaces
-from mantid.simpleapi import MoveInstrumentComponent
-from mantid.simpleapi import RotateInstrumentComponent
-from mantid.simpleapi import CorelliPowderCalibrationApply
+from mantid.simpleapi import (
+    CompareWorkspaces, ConvertToEventWorkspace, CorelliPowderCalibrationApply, CorelliPowderCalibrationCreate,
+    CreateEmptyTableWorkspace, LoadEmptyInstrument, LoadNexus, MoveInstrumentComponent, mtd, RotateInstrumentComponent)
+
+
+class CorelliPowderCalibrationCreateTest(MantidSystemTest):
+    def requiredFiles(self):
+        r"""
+        Runs for standard sample CsLaNb2O7
+        """
+        return ['CORELLI_124036_banks42_87.nxs', 'CORELLI_124036_adjustments.nxs']
+
+    def runTest(self):
+        LoadNexus(Filename='CORELLI_124036_banks42_87.nxs', OutputWorkspace='LaB6')
+        CorelliPowderCalibrationCreate(InputWorkspace='LaB6',
+                                       OutputWorkspacesPrefix='LaB6_',
+                                       TubeDatabaseDir='/SNS/users/jbq/Downloads/calibration_database',
```

Comment:
I'm not sure how the build server can access a user directory on the SNS cluster.  Perhaps changing it into a temp directory where the needed tube dataset sits.

Reply:
you're right. I'll just use `/tmp` because this tests doesn't really use the tube calibration. Anything will do.